src/test/groovy/com/stormpath/tck/oauth2/Oauth2IT.groovy

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2016 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stormpath.tck.oauth2
+
+import com.jayway.restassured.http.ContentType
+import com.stormpath.tck.AbstractIT
+import com.stormpath.tck.util.JwtUtils
+import org.testng.annotations.BeforeClass
+import org.testng.annotations.Test
+
+import static com.jayway.restassured.RestAssured.*
+import static com.jayway.restassured.matcher.RestAssuredMatchers.*
+import static org.hamcrest.Matchers.*
+import static org.testng.Assert.*
+
+@Test
+class Oauth2IT extends AbstractIT {
+
+    private final String randomUUID = UUID.randomUUID().toString()
+    private final String accountEmail = "fooemail-" + randomUUID + "@stormpath.com"
+    private final String accountUsername = randomUUID
+    private final String accountPassword = "P@ssword123"
+
+    private static final String registerRoute = "/register"
+    private static final String tokenRoute = "/oauth/token"
+
+    private String accountHref = ""
+
+    @BeforeClass
+    private void createTestAccount() throws Exception {
+
+        Map<String, Object>  jsonAsMap = new HashMap<>();
+        jsonAsMap.put("email", accountEmail)
+        jsonAsMap.put("password", accountPassword)
+        jsonAsMap.put("givenName", "GivenName-" + randomUUID)
+        jsonAsMap.put("surname", "Surname-" + randomUUID)
+        jsonAsMap.put("username", accountUsername)
+
+        String createdHref =
+            given()
+                .contentType(ContentType.JSON)
+                .body(jsonAsMap)
+            .when()
+                .post(registerRoute)
+            .then()
+                .statusCode(200)
+            .extract()
+                .path("account.href")
+
+        this.accountHref = createdHref
+        deleteOnClassTeardown(createdHref)
```

Comment:
Probably, not entirely sure yet how we want to handle this. Should be a `todo` item or an issue for later work in the TCK.